audio/build.zig

```diff
@@ -4,16 +4,8 @@ const Builder = std.build.Builder;
 const soundio_path = thisDir() ++ "/upstream/soundio";
 
 pub const pkg = std.build.Pkg{
-    .name = "soundio",
-    .source = .{ .path = thisDir() ++ "/soundio/main.zig" },
-};
-
-pub const SoundIoOptions = struct {
-    jack: bool = false,
-    pulseaudio: bool = false,
-    alsa: bool = false,
-    coreaudio: bool = false,
-    wasapi: bool = false,
+    .name = "mach-audio",
```

Comment:
```suggestion
    .name = "audio",
```

I don't know if `mach-audio` is better, but `glfw` and `freetype` are not `mach-glfw` or `mach-freetype` so it seems clear to me this should just be `audio` for consistency sake. Open to changing them all though!